docs/data-sources/equinix_fabric_connection.md

```diff
@@ -261,6 +261,7 @@ Read-Only:
 - `key` (String)
 - `value` (String)
 
+~> **NOTE:** Connection could be modified from Service Provider Side by using parameters passed to additional_info field:  `{"key": "ASN", "value": "1111"}` `{"key": "Global", "value": "false"}` `{"key": "BGP_IBM_CIDR", "value": "172.16.0.18/30"}` `{"key": "BGP_CER_CIDR", "value": "172.16.0.19/30"}`
```

Comment:
This is specifically just for IBM on this one. I know it's a nit, but it should specify IBM side unless we know it applies to all providers.

Reply:
This is also a data source. It isn't making any changes to the underlying connection and is read only. Don't think this information belongs here.